CONTRIBUTING.md

```diff
@@ -93,15 +93,30 @@ This section describes how to configure your development environment to hack on
 
 #### Platform
 
-LavaMoat can comfortably be developed using Linux, macOS, or Windows (but we recommend using WSL on Windows).
+LavaMoat officially supports building on the following operating systems and architectures:
+
+- Latest Ubuntu LTS [`x86_64`]
+- Debian Stable [`x86_64`]
+- Latest Alpine Linux Stable [`x86_64`]
+
+On a best-effort basis, development should also be expected to work on:
+
+- macOS 13.5+ [`x86_64` / `aarch64`]
+- Windows Subsystem for Linux 1 [`x86_64`]
+- Windows Subsystem for Linux 2 [`x86_64` / `aarch64`]
+- Latest Ubuntu LTS [`aarch64`]
+- Debian Stable [`aarch64`]
+- Latest Alpine Linux Stable [`aarch64`]
+
+Developer experience reports for supported and unsupported platforms and compatibility fixes are welcome.
 
 #### Node.js
 
-LavaMoat is distributed as a [Node.js][node] command-line tool. You'll want to install the latest LTS version (see the [Node.js release schedule][release-schedule] for more information).
+LavaMoat is distributed as a [Node.js][node] command-line tool. You'll want to install the active LTS version (see the [Node.js release schedule][release-schedule] for more information).
 
 :::tip
 
-Don't have Node.js installed? You can [download and install Node.js from the official site][node-download], but we recommend using a Node.js version manager (e.g., [nvm][]).
+Don't have Node.js installed? You can [download and install Node.js from the official site][node-download].
```

Comment:
why we switching from recommending a node version manager to not anymore?

Reply:
noting we're introducing

https://github.com/LavaMoat/LavaMoat/blob/98bced7fe99dbf005455c2c0d50632614c9f91d3/Dockerfile#L6

with our current CI matrix range from 16 <--> 22

Reply:
Personally not a fan of the approach as a general recommendation - there are saner ways to manage Node.js runtimes and a specific recommendation feels out of place here.

Reply:
> If anyone relies on containers for development, NVM will also most likely be redundant and (in best case) unused.

true ^

i suppose recommending just use latest official version of node (like [RN on Linux](https://reactnative.dev/docs/set-up-your-environment?os=linux) setup) makes setup easier from the get-go too